Source/Dafny/Dafny.atg

```diff
@@ -1431,6 +1435,7 @@ SynonymTypeDecl<DeclModifierData dmod, ModuleDefinition module, out TopLevelDecl
        }
        td = new OpaqueTypeDecl(id, id.val, module, characteristics, typeArgs, members, attrs, isRefining);
      }
+     td.BodyEndTok = t;
```

Comment:
Same here, the subset type or newtype needs a witness check and I need to know the end of it, until we have proper ranges in nodes.